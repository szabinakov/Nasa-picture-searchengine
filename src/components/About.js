import React from 'react'
import '../styles/About.css'

const About = () => {
    return(
        <div className='aboutContainer'>
            <p className='aboutP'>
            As part of my course at Manchester Codes I have been asked to build a React web application that allows users to search for images based on a query relating to space.<br/> I have used <a className='aboutLink' href='https://api.nasa.gov/'>NASA Images and Video Library</a> for the search functionality.
            </p>
        </div>
    )
}

export default About
