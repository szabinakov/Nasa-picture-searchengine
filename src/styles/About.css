.aboutContainer {
    margin: auto;
    width: 70%;
    border: solid rgb(172, 6, 6);
    padding: 10px;
    background-color: white;
}

.aboutP {
    text-align: justify;
}

.aboutLink {
    text-decoration: underline;
    color: blue;
}

@media screen and (max-width: 500px){

    .aboutContainer {
        width: 80%;
        margin-top: 80px;
    }

    p {
        font-size: 15px;
    }
}