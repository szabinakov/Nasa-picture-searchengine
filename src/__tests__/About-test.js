import React from 'react';
import { render } from '@testing-library/react';
import About from '../components/About.js';

describe('About', () => {

    const {asFragment} = render (<About/>)

    it('it renders correctly', () => {
        
        expect(asFragment()).toMatchSnapshot()
    })
})