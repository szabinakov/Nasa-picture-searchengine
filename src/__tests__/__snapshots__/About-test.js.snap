// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResult it renders correctly 1`] = `
<DocumentFragment>
  <div
    class="aboutContainer"
  >
    <p
      class="aboutP"
    >
      As Part of my course at Manchester Codes I have been asked to build a React web application that allows users to search for images based on a query relating to space.
      <br />
       I have used 
      <a
        class="aboutLink"
        href="https://api.nasa.gov/"
      >
        NASA Images and Video Library
      </a>
       for the search functionality.
    </p>
  </div>
</DocumentFragment>
`;
